### PR TITLE
Auto release via GH Actions

### PR DESCRIPTION
**Pull request by marandaneto**
**What does this PR do?**

Auto release via GH Actions

**Where should the reviewer start?**

**How should this be manually tested?**

**Any background context you want to provide?**

**What are the relevant tickets?**

**Screenshots or screencasts (if UI/UX change)**

**Questions:**
- Does the docs need an update?
- Are there any security concerns?
- Do we need to update engineering / success?

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -1,22 +1,14 @@
...
-```
-
-Then, [configure XCode CLI tools](https://stackoverflow.com/questions/29108172/how-do-i-fix-the-xcrun-unable-to-find-simctl-error).
-
-Finally, prepend `arch -x86_64` to the pod command in the steps above.
+Releasing
+=========
+
+ 1. Update the CHANGELOG.md with the version and date 
+ 2. Go to [GH Releases](https://github.com/PostHog/posthog-android/releases)
```
Comment (benjackwhite) [inline RELEASING.md:5]:
```suggestion
 2. Go to [GH Releases](https://github.com/PostHog/posthog-ios/releases)
```

**Review by benjackwhite — APPROVED** [review]
Overall totally makes sense. I can't say for the actual release flow if it works, but I assume you have that covered :)

File: RELEASING.md (hunk 1 above)
Comment (marandaneto) [reply RELEASING.md:5]: Good catch, copy-pasta mistake

**Comment by marandaneto** [comment]
> Overall totally makes sense. I can't say for the actual release flow if it works, but I assume you have that covered :)

I will try soon, don't trust me :D
Screenshot 2023-10-25 at 08 52 55

if it fails it should be a quick fix anyway, but this is the type of stuff that you cannot test locally, well, I tested the scripts of course :)

**Pull request merged**